www/front_src/src/Header/SwitchThemeMode/index.tsx

```diff
@@ -1,104 +1,73 @@
+/* eslint-disable hooks/sort */
+import { useTransition, useState } from 'react';
+
 import { equals } from 'ramda';
 import { useAtom } from 'jotai';
 import { useLocation } from 'react-router-dom';
+import clsx from 'clsx';
 
-import { styled } from '@mui/material/styles';
-import Switch from '@mui/material/Switch';
+import { Switch, ListItemText } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
+import { grey } from '@mui/material/colors';
 
 import { userAtom, ThemeMode } from '@centreon/ui-context';
 import { patchData, useRequest } from '@centreon/ui';
 
-import svgSun from './images/sun.svg';
-import svgMoon from './images/moon.svg';
-
-interface StyleProps {
-  darkModeSvg?: string;
-  lightModeSvg?: string;
-}
-
-const ThemeModeSwitch = styled(Switch, {
-  shouldForwardProp: (prop) =>
-    !equals(prop, 'color') &&
-    !equals(prop, 'lightModeSvg') &&
-    !equals(prop, 'darkModeSvg'),
-})<StyleProps>(({ theme, darkModeSvg, lightModeSvg }) => ({
-  '& .MuiSwitch-switchBase': {
+const useStyles = makeStyles((theme) => ({
+  container: {
+    '& .MuiSwitch-thumb': {
+      backgroundColor: 'white',
+    },
+    '& .MuiSwitch-track': {
+      backgroundColor: '#aab4be',
```

Comment:
Why do you need to customize colors here?

Reply:
without customizing color :